src/Components/Common/OnlineUsersSelect.tsx

```diff
@@ -1,37 +1,47 @@
-import loadable from "@loadable/component";
 import { CircularProgress } from "@material-ui/core";
-import React, {
-  useCallback,
-  useReducer,
-  useState,
-  useEffect,
-  useRef,
-} from "react";
+import React, { useCallback, useState, useEffect } from "react";
 import { useDispatch } from "react-redux";
 import { statusType, useAbortableEffect } from "../../Common/utils";
 import moment from "moment";
 import { getUserList } from "../../Redux/actions";
 import classNames from "classnames";
 
-export const OnlineUsersSelect = (props: any) => {
+type State = {
+  loading: boolean;
+  isExpanded: boolean;
+  users: Array<any>;
+  searchTerm: string;
+  searchFieldRef: React.RefObject<HTMLInputElement>;
+};
+
+type Props = {
+  selectedUser: any;
+  userId: string;
+  onSelect: any;
+  user_type: string;
+};
+
+const initialState: State = {
+  loading: false,
+  isExpanded: false,
+  users: new Array<any>(),
```

Comment:
same with this we can have a user model, if it isn't present right now... and replace the instances if any.